### DISCUSSION TITLE
Merge Queue feature availability?

### DISCUSSION THREAD

**Discussion by oerp-odoo** [community/community#51483] [Repositories] [answered]
I recently created public repository and noticed that there is this new option under branch protection rules:

![Selection_468](https://user-images.githubusercontent.com/7812986/228819019-f23f4e3d-57d0-4030-be4a-52bd78cd9a3d.png)

But if I go my private repository (under organization), there is no such feature. 

Does it mean, its currently only available for public repositories? Or maybe there is a way to opt in for this feature for any repo?

**Comment by u-ways** [comment, marked as answer]
Yes, you will need to be in private repositories owned by organizations using GitHub Enterprise Cloud.

> Pull request merge queues are available in any public repository owned by an organization, or in private repositories owned by organizations using GitHub Enterprise Cloud. For more information, see "[GitHub’s products](https://docs.github.com/en/get-started/learning-about-github/githubs-products)."

See: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue

**Comment by oerp-odoo** [comment]
Strange that non enterprise orgs are excluded:)

**Comment by zaibon** [comment]
Are there any plans to include non-enterprise organizations at some point, or will this feature never be available for them?

**Comment by anguslees** [comment]
I've read https://github.blog/news-insights/product-news/github-merge-queue-is-generally-available/ many times

> Any team that is part of a managed organization with public repositories and GitHub Enterprise Cloud users will be able to enable this feature on their respective repository and start streamlining their team’s pull requests immediately. If this describes you, then let the commit parties commence

I think a more correct title for this blog is "GitHub merge queue is _not_ generally available."  Have I overlooked something?

**Comment by Balram72** [comment]
I think a more correct title for this blog is "GitHub merge queue is not generally available." Have I overlooked something

**Comment by Balram72** [comment]
Are there any plans to include non-enterprise organizations at some point, or will this feature never be available for them?

**Comment by sabjorn** [comment]
will this feature ever be opened up for GitHub Team for private repos?

**Comment by tanvishinde017** [comment]
What you’re seeing is most likely related to newer “Repository Rules / Rulesets” features in GitHub, not the classic branch protection itself.

👉 These newer options are not always available everywhere, and that’s why you see them in your public repo but not in your private org repo.

**Comment by Gravifer** [comment]
Is it possible to open up this feature to public repos under free personal accounts?

Despite receiving recent pro-agent moves of the platform negatively, I do think in light of multi-agent workflows, democratizing NotRocketScience is would be a net benefit.
